helm/values.yaml

```diff
@@ -34,6 +34,10 @@ containerPorts:
     containerPort: 14250
   - name: http-zipkin
     containerPort: 9411
+  - name: http-promint
+    containerPort: 8888
+  - name: http-promexp
+    containerPort: 8889
```

Comment:
what does `promint` and `promexp` mean?

Reply:
Prometheus_internal_metrics, Prometheus_exporter_metrics

Reply:
Please avoid abbreviations to avoid any confusion. ports can be named as `http-prom-internal` and `http-prom-exporter`

Reply:
Adding a comment is also helpful.

Reply:
containerPorts need to be less than 15 chars. So added comments and renamed to `http-prom-int` and `http-prom-exp`